app/assets/javascripts/admin/addon/components/admin-config-area-card.gjs

```diff
@@ -0,0 +1,16 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class AdminConfigAreaCard extends Component {
+  @tracked collapsed = false;
+
+  <template>
+    <section class="admin-config-area-card">
+      <h3>{{i18n @heading}}</h3>
+      <form>
```

Comment:
I _think_ we may want to just keep this as a `div`, maybe with `admin-config-area-card__content` CSS class. We want people using it to have flexibility, though you are right in that most of the time it's likely going to be a form.

Reply:
done!